tests/Chain/ToolBox/OpenMeteoTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpLlm\LlmChain\Tests\Chain\ToolBox;
+
+use PhpLlm\LlmChain\Chain\ToolBox\Tool\OpenMeteo;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\Test;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpClient\MockHttpClient;
+use Symfony\Component\HttpClient\Response\JsonMockResponse;
+
+#[CoversClass(OpenMeteo::class)]
+final class OpenMeteoTest extends TestCase
+{
+    #[Test]
+    public function current(): void
+    {
+        $response = $this->jsonMockResponseFromFile(__DIR__.'/fixtures/openmeteo-current.json');
+        $httpClient = new MockHttpClient($response);
+
+        $openMeteo = new OpenMeteo($httpClient);
+
+        $actual = $openMeteo->current(52.52, 13.42);
+        $expected = [
+            'weather' => 'Overcast',
+            'time' => '2024-12-21T01:15',
+            'temperature' => '2.6°C',
+            'wind_speed' => '10.7km/h',
+        ];
+
+        static::assertSame($expected, $actual);
+    }
+
+    #[Test]
+    public function forecast(): void
+    {
+        $response = $this->jsonMockResponseFromFile(__DIR__.'/fixtures/openmeteo-forecast.json');
```

Comment:
JsonMockResponse::fromFile() is not available?

Reply:
got only added with v7.1 - so would break with lowest symfony version